src/components/FileList/FileList.js

```diff
@@ -20,38 +20,35 @@ const FileList = ({fileList, studyId}) => {
   const handlePageClick = (e, {activePage}) => {
     setPage(activePage);
   };
-  const pageCount = Math.ceil(fileList.length / perPage);
-  const pageItems = fileList.slice(
+  var fileListCopy = JSON.parse(JSON.stringify(fileList));
+  var statusList = [];
+  var i;
+  for (i in fileListCopy) {
```

Comment:
I think you may be able to do this with the `sort` function:
```
fileListCopy.sort((el1, el2) => {
    let el1Status = fileSortedVersions[0].node.state;
    return el1Status === "CHN" ? 1 : 0
})
```

Reply:
👍